orttraining/orttraining/python/ort_trainer.py

```diff
@@ -912,6 +987,8 @@ def train_step(self, *args, **kwargs):
         if not isinstance(input, (list, tuple)):
             input = (input,)
 
+        if self._debug_conversion and self.current_step is 1: self._debug_conversion_step(input, input_descs, output_desc)
```

Comment:
what should happen after `_debug_conversion_step`? I believe we should end the script as the user will probably want to look the result and not continue the training.

Let's double check that requirement

Reply:
`self.current_step == 1` is recommended for `int` comparison